### DISCUSSION TITLE
Resuggesting if I want a lsp server

### DISCUSSION THREAD

**Discussion by bwsdm** [VonHeikemen/lsp-zero.nvim#109] [Q&A] [answered]
How would I go about getting lsp-zero to resuggest a lsp server? My terminal flickered and I didnt get the opportunity to select yes when it popped up. Now it doesnt ask me anymore for that filetype. Thanks in advance

In case I need to clarify, its not lsp-zero's fault that my terminal flickered. My setup at work is a bit janky on windows since there are alot of things locked down

**Comment by VonHeikemen** [comment, marked as answer]
The server suggestions are available using the command `:LspInstall`.
